### DISCUSSION TITLE
citation key field output empty

### DISCUSSION THREAD

**Discussion by crispysmiley** [retorquere/zotero-better-bibtex#2707] [Q&A] [answered]
I'm trying to the citation key in my **Zotcard** customized template with `${citationKey}`, which is a built-in field in Zotcard, but the output is null. I have **Better BibTex** installed and can find the "citation key" field in the item info, it is not null. 

I also tried the official `${topItem.citationKey ? topItem.citationKey : ""}` in **Better Notes**, but that doesn't output anything either. Therefore, I guess there is something wrong with my Bibtex settings. Can anyone help to see why this is?

My Zotcard template reads like this: 

```
<h1 style="font-size:12pt">${title}</h1>
<p>${authors}, ${year}</p>
<p><strong>Citation Key:</strong>${citationKey}</p>

<h2>Research Question</h2>

<p></p>

<h2>Important Points for me</h2>

<p></p>

<h2>Contributions for my topic</h2>

<p></p>

<h2 style="font-size:13pt">Limits</h2>

<p></p>
```

**Comment by retorquere** [comment, marked as answer]
You'll have to ask the authors of those plugins, I don't know how they work.
